requirements.txt

```diff
@@ -2,6 +2,7 @@ configparser==3.5.0
 docopt==0.6.2
 file-magic==0.3.0
 jsonschema==2.5.1
+libarchive-c==2.5
```

Comment:
Yes please. I think that your PR will land before mine, because I still have work to do on the snapcraft/debian branch.